### PR TITLE
feat: Connect Register to server

### LINKED ISSUE #15: feat: Connect Register to server

### PR DESCRIPTION
**Pull request by marimarimarietty**

### CODE REVIEW

File: src/components/molecules/Signup.tsx [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+      zip_code: data.zipcode,
+      serial_number: data.serialnumber,
+      password: data.password,
+    });
+
+    const customConfig = {
+      headers: {
+        "Content-Type": "application/json",
+      },
+    };
```
Comment (Takashicc) [inline src/components/molecules/Signup.tsx]: Do you need this config?
I think Axios will automatically handle json.

File: src/components/molecules/Signup.tsx [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+
+    const formData = new FormData();
+    formData.append("username", data.email);
+    formData.append("password", data.password);
```
Comment (Takashicc) [inline src/components/molecules/Signup.tsx]: Move these lines to the line 186

**Review by Takashicc — CHANGES_REQUESTED** [review]

File: src/components/molecules/Signup.tsx (hunk 1 above)
Comment (marimarimarietty) [reply src/components/molecules/Signup.tsx]: After trying and error, this description seems to be necessary.

Comment (Takashicc) [inline src/components/molecules/Signup.tsx]: Don't wrap with `JSON.stringify` and delete the `customConfig`.

**Review by Takashicc — CHANGES_REQUESTED** [review]

**Review by Takashicc — APPROVED** [review]

**Pull request merged**